cellprofiler/modules/measureimageareaoccupied.py

```diff
@@ -119,11 +119,12 @@ def __init__(self):
                 self.__should_save_image = cellprofiler.setting.Binary(
                     "Retain a binary image of the object regions?",
                     False,
-                    doc="""
-                    <i>(Used only if '{O_OBJECTS}' are to be measured)</i><br>
-                    Select <i>{YES}</i> if you would like to use a binary image later in the pipeline, for example in
-                    <b>SaveImages</b>. The image will display the object area that you have measured as the foreground
-                    in white and the background in black.
+                    doc="""\
+*(Used only if ‘{O_OBJECTS}’ are to be measured)*
```

Comment:
Add a newline after *(Used only if...)* text to insert a line break in the documentation rendered in the GUI.